src/Components/NavBar/NavBar.tsx

```diff
@@ -1,18 +1,22 @@
 import React from 'react';
 import './NavBar.css';
 
-const NavBar = () => {
+interface LinkProps {
+  retrieveBooks: Function;
+}
+
+const NavBar: React.FC<LinkProps> = ({ retrieveBooks }) => {
 
   return (
     <nav className="sidenav">
-      <button className="biography">Biography</button>
-      <button className="children">Children</button>
-      <button className="fiction">Fiction</button>
-      <button className="history">History</button>
-      <button className="nonfiction">Nonfiction</button>
-      <button className="poetry">Poetry</button>
-      <button className="romance">Romance</button>
-      <button className="self-help">Self-Help</button>
+      <button className="biography" onClick={() => retrieveBooks("biography")}>Biography</button>
```

Comment:
Possible refactor, to pass through the className or create an ID for each of the elements...